apps/dot_ext/management/commands/authflow_id_delete_and_vacuume.py

```diff
@@ -0,0 +1,67 @@
+import time
+from django.db import connection
+
+from datetime import date, timedelta
+from django.core.management.base import BaseCommand
+
+from apps.dot_ext.models import AuthFlowUuidCopy
+
+DELETE_WITH_LIMIT = "DELETE FROM {table_name} WHERE auth_uuid IN (SELECT auth_uuid FROM {table_name} WHERE created < '{age_date}'::date ORDER BY created LIMIT {limit_on_delete})"
+VACUUM_TABLE = "VACUUM FULL {table_name}"
+
+
+def delete_and_vacuume_auth_uuid_table(age, limit_on_delete, run_vacuum, dry_run):
+    '''
+    Delete aged authflow uuid records, and then
+    call VACUUM FULL (Postgres only) on table: dot_ext_authflowuuid
+    '''
+
+    days_before = date.today()-timedelta(days=age)
+    # change table name dot_ext_authflowuuidcopy to dot_ext_authflowuuid before merge
+    delete_sql = DELETE_WITH_LIMIT.format(table_name="dot_ext_authflowuuidcopy", age_date=days_before.strftime("%m/%d/%Y"), limit_on_delete=limit_on_delete)
+    vacuum_sql = VACUUM_TABLE.format(table_name="dot_ext_authflowuuidcopy")
+
+    start_t = time.time()
+
+    aged_records_total = AuthFlowUuidCopy.objects.filter(created__lt=days_before).count()

```

Comment:
I think maybe we should just pass on this up front. It'll be time consuming while the table is large. You also should get the number of rows deleted in the output. If it runs fast though it might be interesting information to keep in. What's the run time look like?

Reply:
Maybe we only display this on the dry run? Save time on the real run?